tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1466,6 +1466,30 @@ RDataFrame::RDataFrame(std::unique_ptr<ROOT::RDF::RDataSource> ds, const ColumnN
 /// spec.AddFriend([("tree1", "a.root"), ("tree2", "b.root")], "alias")
 /// df = ROOT.RDataFrame(spec)
 /// ~~~
+///
+/// ~~~{.cpp}
+/////First, create a dummy root file with two trees,
```

Comment:
The example code should have a whitespace between the three forward slashes that are needed for doxygen and the rest of the body, as found in lines above